lib/load_brocfile.ts

```diff
@@ -1,8 +1,5 @@
 import path from 'path';
 import findup from 'findup-sync';
-import esm from 'esm';
```

Comment:
this is the main fix here. importing `esm` _cannot_ happen in module-space because of the hax it employs.